pkg/skaffold/docker/dependencies.go

```diff
@@ -0,0 +1,173 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package docker
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/docker/docker/builder/dockerignore"
+	"github.com/docker/docker/pkg/fileutils"
+	"github.com/karrick/godirwalk"
+	"github.com/pkg/errors"
+)
+
+// NormalizeDockerfilePath returns the absolute path to the dockerfile.
+func NormalizeDockerfilePath(context, dockerfile string) (string, error) {
```

Comment:
this was copied from `pkg/skaffold/docker/parse.go` right? can we not move it to a common location (`pkg/skaffold/docker/util/util.go`?) and reuse it?

Reply:
@nkubala So the function was not copied but moved. It is therefore already reused.

I'm not so happy with moving it to `util`. Such packages/files quickly attract completely unrelated code. But if you feel strongly about it, I won't start to argue.